Confus/Confus/Game.h

```diff
@@ -96,7 +94,7 @@ namespace Confus
         /// <summary>
         /// Finalizes an instance of the <see cref="Game"/> class.
         /// </summary>
-        virtual ~Game() = default;
+        ~Game();
```

Comment:
Though it's not important right now, it is not a bad idea to keep this virtual. The main reason for this has to do with inheriting from the game class and having your own destructor. Consider deriving MainGame from Game. I store an instance of MainGame in a Game pointer, however, once we get rid of the Game instance (for example by using delete), it wouldn't know MainGame's destructor still needs to be called (because it isn't marked virtual).